src/js/components/RadioButtonGroup/doc.js

```diff
@@ -61,3 +61,20 @@ export const doc = RadioButtonGroup => {
 
   return DocumentedRadioButtonGroup;
 };
+
+export const themeDoc = {
+  'radioButtonGroup.gap': {
+    description: `The amount of spacing between child elements.`,
+    type: 'string',
+    defaultValue: `{
+      none,
+      xxsmall,
+      xsmall,
+      small,
+      medium,
+      large,
+      xlarge,
+      any css size,
+    }`,
```

Comment:
I don't think the default value is an object, it should be `undefined`.

Reply:
And regardless, the docs need to accommodate your recent changes where all the Box props are included and not a single doc entry just for gap prop.

Reply:
something like this:
```
export const themeDoc = {
  'radioButtonGroup.a11yTitle': {
    description: `a user friendly label for screen readers`,
    type: 'string',
    defaultValue: `undefined`,
  },
'radioButtonGroup.align': {
    description: `How to align the contents along the cross axis.`,
    type: 'string',
    defaultValue: `undefined`,
  }
  ...
  ...
};
```

am i correct to assume that, if there is no bolded value in the docs it defaults to undefined?

Reply:
Similar to https://v2.grommet.io/select#select.options.container